src/core/dep/jquery-fix.js

```diff
@@ -163,6 +163,33 @@ jQuery.htmlPrefilter = function( html ) {
  * This implementation leverage DOMPurify for filtering every string prior DOM manipulation by jQuery
  *
  */
+
+// START: add hooks to DOMPurify to allow external links when they meet certain conditions as defined here: https://owasp.org/www-community/attacks/Reverse_Tabnabbing
+DOMPurify.addHook( "beforeSanitizeAttributes", function( node ) {
+
+	// Add "data-external-link" to all <a> with a target="_blank" and rel="noreferrer"
+	if (
+		node.tagName === "A" &&
+		node.getAttribute( "target" ) &&
+		node.getAttribute( "target" ) === "_blank" &&
+		node.getAttribute( "rel" ) &&
+		node.getAttribute( "rel" ).includes( "noreferrer" )
+	) {
+		node.setAttribute( "data-external-link", "true" );
+	}
+} );
+
+DOMPurify.addHook( "afterSanitizeAttributes", function( node ) {
+
+	// Put back the target="_blank" to all <a> with attribute "data-external-link"
+	if ( node.getAttribute( "data-external-link" ) ) {
```

Comment:
I suggest to pre-check the tagName as this is only applicable to anchor element. Calling a function which need to iterate over each attribute should take more computation time compared to check a static property. That way we reduce the number iteration on non-anchor element.

```suggestion
	if ( node.tagName === "A" && node.getAttribute( "data-wb-external-link" ) ) {
```